gcm/gcm_connector.go

```diff
@@ -35,45 +40,74 @@ func NewGCMConnector(router server.Router, prefix string, gcmAPIKey string) (*GC
 		return nil, err
 	}
 
+	//TODO Cosmin: check with dev-team the number of GCM workers, below
 	gcm := &GCMConnector{
 		router:            router,
 		kvStore:           kvStore,
 		prefix:            prefix,
 		channelFromRouter: make(chan server.MsgAndRoute, 1000),
 		stopChan:          make(chan bool, 1),
 		sender:            &gcm.Sender{ApiKey: gcmAPIKey},
+		workersNumber:     runtime.GOMAXPROCS(0),
 	}
 
 	return gcm, nil
 }
 
-// Start opens the connector and awaits for messages from router to be forwarded to gcm until the stop signal is emitted
+// Start opens the connector, start more goroutines / workers to handle messages coming from the router
 func (conn *GCMConnector) Start() error {
 	broadcastRoute := server.NewRoute(removeTrailingSlash(conn.prefix)+"/broadcast", conn.channelFromRouter, "gcm_connector", "gcm_connector")
```

Comment:
Would it be a good idea to store this broadcast path in the struct? 
We wont have to build it every use.